### ISSUE TITLE
Bug: Read Time Out

### ISSUE THREAD

**Issue by davep76** [awslabs/multi-agent-orchestrator#46]
### Expected Behaviour

It should not timeout

### Current Behaviour

ERROR:multi_agent_orchestrator.utils.logger:Error during agent dispatch or processing:AWSHTTPSConnectionPool(host='[bedrock-agent-runtime.us-east-1.amazonaws.com](http://bedrock-agent-runtime.us-east-1.amazonaws.com/)', port=443): Read timed out.

### Code snippet

```python
I am using BR agent with 5 AG
```

### Possible Solution

Increase the timeout limit

### Steps to Reproduce

Create a BR Agent with 5-6 AGs

**Comment by brnaba-aws** [comment]
We will look into this.

**Comment by brnaba-aws** [comment]
@davep76 we have released the 0.0.14 version, can you `pip install multi-agent-orchestrator==0.0.14`

and then when creating your bedrock agent, you can pass a client.
```python
my_agent = AmazonBedrockAgent(
        AmazonBedrockAgentOptions(
            name="my-agent",
            description="Agent description",
            agent_id="AGENT-ID",
            agent_alias_id="AGENT-ALIAS-ID",
            client=boto3.client('bedrock-agent-runtime', config=your-config)
        )
    )
```
